CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## CHANGELOG
 
+### 3.X.X (2021-MM-DD)
```

Comment:
3.6.0 hasn't been released yet, so this should go under that section. And probably 3.6.0 should be renamed to 3.X.X like you did here.

@munir-rakuten Looks like there are also some other problems with the Changelog: `admob-latest` wasn't released in 3.5.0, right? I think that was merged after the release.

Reply:
Yeah, I have updated the changelog for upcoming release.

@corycaywood @munir-rakuten I think, `admob-latest` was included in 3.5.0 e.g. https://github.com/rakutentech/android-miniapp/compare/v3.5.0...master 🤔